ui/src/components/admin/faqs/AdminFaqContainer.js

```diff
@@ -94,7 +94,7 @@ export default function AdminFaqContainer(props) {
         pageTopic: pageTopic,
         question: question,
         answer: answer,
-        isPublished: published,
+        order: order === '' ? -1 : order,
```

Comment:
realized this has no duplicate order checking so we might have duplicate orders. not sure how necessary to fix since things look like it's working ok

<img width="313" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/16095743/22608e8a-a68d-43ad-88be-7b2d0bead56f">

Reply:
realized i didn't add a check for the order update through the grid (i.e. if FAQ is being unpublished, set order to -1), i think should work fine after i add it